lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -330,6 +331,7 @@ WasmBinaryReader::ASTNode()
         BlockNode();
         break;
     case wbCall:
+    case wbCallImport:
```

Comment:
I added this as tmp measure, but will need some convergence with import table change